evidence/koute/0001-koute-evidence-2024-05-09.md

```diff
@@ -0,0 +1,34 @@
+# Evidence-0001: Promotion to Rank III
+
+|                  |                      |
+| ---------------- | ---------------------|
+| **Report Date**  | 2024/05/09           |
+| **Submitted by** | Jan Bujak            |
+
+## Member details
+
+- Matrix username: @jan:parity.io
+- Polkadot address: 15tRXfXoZXkjScB3Awv8s2saPjaicKYAyL1WZ3JP5kycoG9n
+- Current rank: 2
+- Date of initial induction: 2024/01/08
```

Comment:
Jan was part of the Technical Fellowship [seeding](https://github.com/polkadot-fellows/seeding).
So, the Date of initial induction should read "seeding".

Reply:
Done; thanks.